docs/pages/connect-your-client/teleport-connect.mdx

```diff
@@ -158,27 +158,71 @@ installation directory to the `Path` user environment variable.
+| `usageReporting.enabled`      | `false`                                                                                                              | Enables collecting anonymous usage data (see [Telemetry](#telemetry)). |
+| `keymap.tab1` - `keymap.tab9` | `Command+1` - `Command+9` on macOS <br/> `Ctrl+1` - `Ctrl+9` on Windows<br/>`Alt+1` - `Alt+9` on Linux               | Shortcut to open tab 1–9.                                              |
+| `keymap.closeTab`             | `Command+W` on macOS<br/>`Ctrl+W` on Windows/Linux                                                                   | Shortcut to close a tab.                                               |
+| `keymap.newTab`               | `Command+T` on macOS<br/>`Ctrl+T` on Windows/Linux                                                                   | Shortcut to open a new tab.                                            |
+| `keymap.previousTab`          | `Shift+Command+Tab` on macOS<br/>`Ctrl+Shift+Tab` on Windows/Linux                                                   | Shortcut to go to the previous tab.                                    |
+| `keymap.nextTab`              | `Command+Tab` on macOS<br/>`Ctrl+Tab` on Windows/Linux                                                               | Shortcut to go to the next tab.                                        |
+| `keymap.openConnections`      | `Command+P` on macOS<br/>`Ctrl+P` on Windows/Linux                                                                   | Shortcut to open the connection list.                                  |
+| `keymap.openClusters`         | `Command+E` on macOS<br/>`Ctrl+E` on Windows/Linux                                                                   | Shortcut to open the cluster selector.                                 |
+| `keymap.openProfiles`         | `Command+I` on macOS<br/>`Ctrl+I` on Windows/Linux                                                                   | Shortcut to open the profile selector.                                 |
+| `keymap.openCommandBar`       | `Command+K` on macOS<br/>`Ctrl+K` on Windows/Linux                                                                   | Shortcut to open the command bar.                                      |
+| `terminal.fontFamily`         | `Menlo, Monaco, monospace` on macOS<br/>`Consolas, monospace` on Windows<br/>`'Droid Sans Mono', monospace` on Linux | Font family for the terminal.                                          |
+| `terminal.fontSize`           | 15                                                                                                                   | Font size for the terminal.                                            |
+
+<Admonition
+  type="note"
+  title="Note"
+>
+  The additional `$schema` property present in the config file allows text editors to provide autocompletion.
```

Comment:
This is the only time we mention the `$schema` property, so I think it would be good to include an example of how to use it and when it would be useful.

Reply:
Technically, it should work out of the box, but I can add something like this: 
`It is a JSON schema file that describes the config properties. Your editor should recognize it automatically.`.

WDYT?

Reply:
Just to clarify, when you install Teleport Connect, does it write a config file automatically? And does that config file include the `$schema` property? 

If so, we might want to clarify that users don't need to create their own config file, that the `$schema` property exists in the default config file, and that users should not change it.

Reply:
>when you install Teleport Connect, does it write a config file automatically? And does that config file include the $schema property?

Yes and yes.

>If so, we might want to clarify that users don't need to create their own config file

The paragraph starts with:
```
Teleport Connect can be configured by editing the `app_config.json` file.
To open it, click the More Options icon `⋮` -> Open Config File.
```
IMO it is clear that the file has been already created, isn't it? 

> ...that the $schema property exists in the default config file, and that users should not change this property.

I added a line saying that it should not be modified.